docs/source/micromamba-installation.rst

```diff
@@ -126,8 +126,7 @@ An exclusive `conda-forge <https://conda-forge.org/>`_ setup can be configured w
 
 .. code-block:: bash
 
-   micromamba config append channels conda-forge
-   micromamba config append channels nodefaults
+   micromamba config append channels conda-forge`
```

Comment:
This works only until https://github.com/mamba-org/mamba/issues/2758 is fixed. After that we need to set `default_channels: []` but that doesn't work because of https://github.com/mamba-org/mamba/issues/2140

Reply:
It's a no-op